test/unit/slave/test_subjob_executor.py

```diff
@@ -47,14 +49,14 @@ def test_execute_subjob_passes_correct_build_executor_index_to_execute_command_i
         atomic_commands = ['command']
         executor.id = 2
         expected_env_vars = {
-            'ARTIFACT_DIR': 'path',
+            'ARTIFACT_DIR': '/tmp/1/artifact_2_0',
```

Comment:
Almost certainly no. Usually, I would do
`Configuration['artifact_directory'] = os.path.expanduser('~')`

`'ARTIFACT_DIR': join(Configuration['artifact_directory'], '1', 'artifact_2_0')`.

Thanks for catching it, Joey!